docs/src/inputs/select/examples/exampleCreatableSelect.jsx

```diff
@@ -0,0 +1,32 @@
+import { isEmpty } from 'lodash';
+import React, { useState } from 'react';
+import {
+    Select,
+} from 'react-cm-ui';
+import { options } from '../constants';
+
+function ExampleCreatableSelect() {
+    const [selectedValue, setOption] = useState({});
+
+    function onSelect(selectedOption) {
+        setOption(selectedOption);
+    }
+
+    return (
+        <div>
+            <Select
+                clearable
+                creatable
+                id="block--creatable_select_id"
+                options={options}
+                onChange={onSelect}
+                placeholder="Select Option"
+                searchable
+                selection
```

Comment:
Same here: I don't think we need `selection` prop.  Please remove wherever it is not needed.

Reply:
@groberts314 Thank you for catching it. I have corrected it.

Reply:
Gonezo; resolving.